versions/datastax/3.24.0/patch.patch

```diff
@@ -0,0 +1,156 @@
+ import shutil
+ 
++
+ from cassandra import OperationTimedOut, ReadTimeout, ReadFailure, WriteTimeout, WriteFailure, AlreadyExists,\
+     InvalidRequest
+ from cassandra.protocol import ConfigurationException
+@@ -44,6 +45,7 @@
+ try:
+     from ccmlib.dse_cluster import DseCluster
+     from ccmlib.cluster import Cluster as CCMCluster
++    from ccmlib.scylla_cluster import ScyllaCluster as CCMScyllaCluster
+     from ccmlib.cluster_factory import ClusterFactory as CCMClusterFactory
+     from ccmlib import common
+ except ImportError as e:
+@@ -161,16 +163,21 @@
+ SIMULACRON_JAR = os.getenv('SIMULACRON_JAR', None)
+ CLOUD_PROXY_PATH = os.getenv('CLOUD_PROXY_PATH', None)
+ 
+-# Supported Clusters: Cassandra, DDAC, DSE
++# Supported Clusters: Cassandra, DDAC, DSE, Scylla
+ DSE_VERSION = None
++SCYLLA_VERSION = os.getenv('SCYLLA_VERSION', None)
+ if os.getenv('DSE_VERSION', None):  # we are testing against DSE
+     DSE_VERSION = Version(os.getenv('DSE_VERSION', None))
+     DSE_CRED = os.getenv('DSE_CREDS', None)
+     CASSANDRA_VERSION = _get_cass_version_from_dse(DSE_VERSION.base_version)
+     CCM_VERSION = DSE_VERSION.base_version
+ else:  # we are testing against Cassandra or DDAC
+-    cv_string = os.getenv('CASSANDRA_VERSION', None)
+-    mcv_string = os.getenv('MAPPED_CASSANDRA_VERSION', None)
++    if SCYLLA_VERSION:
++        cv_string = SCYLLA_VERSION
++        mcv_string = os.getenv('MAPPED_SCYLLA_VERSION', None)
++    else:
++        cv_string = os.getenv('CASSANDRA_VERSION', None)
++        mcv_string = os.getenv('MAPPED_CASSANDRA_VERSION', None)
+     try:
+         cassandra_version = Version(cv_string)  # env var is set to test-dse for DDAC
+     except:
+@@ -435,7 +442,7 @@
+         if [len(list(nodes)) for dc, nodes in
+                 groupby(CCM_CLUSTER.nodelist(), lambda n: n.data_center)] == node_counts:
+             for node in CCM_CLUSTER.nodelist():
+-                if set(node.workloads) != set(workloads):
++                if set(getattr(node, 'workloads', [])) != set(workloads):
+                     print("node workloads don't match creating new cluster")
+                     return False
+             return True
+@@ -556,8 +563,15 @@
+ 
+                 CCM_CLUSTER.set_dse_configuration_options(dse_options)
+             else:
+-                CCM_CLUSTER = CCMCluster(path, cluster_name, **ccm_options)
+-                CCM_CLUSTER.set_configuration_options({'start_native_transport': True})
++                if SCYLLA_VERSION:
++                    # `experimental: True` enable all experimental features.
++                    # CDC is causing an issue (can't start cluster with multiple seeds)
++                    # Selecting only features we need for tests, i.e. anything but CDC.
++                    CCM_CLUSTER = CCMScyllaCluster(path, cluster_name, **ccm_options)
++                    CCM_CLUSTER.set_configuration_options({'experimental_features': ['lwt', 'udf'], 'start_native_transport': True})
```

Comment:
That shouldn't be the case anymore, did you try it?
Who wrote this part? is it you?

Reply:
This is the code from our python-driver repo.
We run this matrix for the following version:
* scylla-4.4
*scylla-4.5
* scylla-4.6
* master
Do you want me to remove it?

Reply:
I've put those `experimental_features`, a long long time ago. (when lwt was still experimental)

Reply:
> I've put those `experimental_features`, a long long time ago. (when lwt was still experimental)

Yes, this change is very old (from 2020).
Is LWT not experimental for Scylla 4.4?

Reply:
I don't think so, you can remove.